images/basic/fuse/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN set -x \
+    && apt-get update \
+    && apt-get install -y \
+    wget \
+    git \
+    pkg-config \
+    zip \
+    g++ \
+    zlib1g-dev \
+    unzip \
+    python3 \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get install -y software-properties-common
+
+RUN apt update
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test
+RUN apt-get install -y build-essential git pkg-config fuse3 libfuse3-3 libfuse3-dev strace
+RUN apt-get install -y procps vim libstdc++6
+
+WORKDIR /fus
+
+RUN mkdir -pv mountpoint
+COPY server-bin ./server-bin
```

Comment:
Why don't we build binary in this container? Pushing a binary file in git source tree is usually not a good idea.